algorithms/kernel/k_nearest_neighbors/bf_knn_impl.i

```diff
@@ -61,8 +61,8 @@ public:
         const size_t nTrain = trainTable->getNumberOfRows();
         const size_t nTest  = testTable->getNumberOfRows();
 
-        int * trainLabel = nullptr;
-        BlockDescriptor<int> trainLabelBlock;
+        FPType * trainLabel = nullptr;
```

Comment:
```suggestion
        FPType * trainLabel = NULL;
```
If I'm not mistaken `nullptr` is prohibited since it's part of C++ library

Reply:
We use nullptr everywhere, it's not a part of std